File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;
class Solution {
    public int[] solution(int[] numbers) {
        int[] answer = {};
        Arrays.sort(numbers);
        
        List<Integer> list = new ArrayList<>();

        for(int i = 0; i < numbers.length; i++){
            for(int j = i + 1; j < numbers.length; j++){
                int temp = numbers[i] + numbers[j];
                if(!list.contains(temp)){
                    list.add(temp);
                }
            }
        }
        
        answer = new int[list.size()];
        
        for(int k = 0; k < list.size(); k++){
            answer[k] = list.get(k);
        }
        
        Arrays.sort(answer);
        
        return answer;
    }
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []
    length = len(numbers)
    
    for i in range(0, length):
        for j in range(i + 1, length):
            n = numbers[i] + numbers[j]
            if n not in answer:
                answer.append(n)

    answer.sort()

    return answer